### PR TITLE
Updated macOS specific GAMS installation guidelines

### PR DESCRIPTION
**Pull request by OFR-IIASA**
This PR updates the installation for installing GAMS on macOS.

## How to review

- Ensure the modified page is rendered correctly.

## PR checklist

- [x] Continuous integration checks all ✅

- [x] Add or expand tests; coverage checks both ✅
- [x] Add, expand, or update documentation.
- [x] Update release notes.

### CODE REVIEW

File: RELEASE_NOTES.rst [hunk 1]
```diff
@@ -4,6 +4,7 @@ Next release
 All changes
 -----------
 
+- :pull:`460`: Expand documentation :doc:`INSTALL` for installing GAMS under macOS.
```
Comment (khaeru) [inline RELEASE_NOTES.rst:7]: Can you double-check that this link works when the docs are built? Although the file is at the project root in INSTALL.rst, it's included in the documentation via doc/install.rst, so I suspect the lower case is needed within this link.

**Review by khaeru — CHANGES_REQUESTED** [review]
Thanks @OFR-IIASA for this refinement! One minor check and possible change, then this can be merged.

File: RELEASE_NOTES.rst (hunk 1 above)
Comment (OFR-IIASA) [reply RELEASE_NOTES.rst]: Thanks @khaeru. I have corrected and checked that the link works.

**Review by khaeru — APPROVED** [review]
Approved!

Will squash and merge because of the [commit message format](https://docs.messageix.org/en/stable/contributing.html#code-style).

**Pull request merged**